spec/shackles_spec.rb

```diff
@@ -4,6 +4,7 @@
 require 'shackles'
 
 # we're not actually bringing up ActiveRecord, so we need to initialize our stuff
+Shackles.primary_environment_name = :primary
```

Comment:
Is this the right usage for configuration this? I'm not familiar enough with railties to know how this would really work in a Rails application w.r.t. where/when you would run this line.

Note that this doesn't test the default case of `:master`. I had trouble testing both those code paths in this same file given that everything gets initialized once on the next line. I'm open to other suggestions or changing it back and _just_ testing the default case without specifying a `primary_environment_name`.

Reply:
just default it to primary, and not worry about it.